frame/support/src/traits/metadata.rs

```diff
@@ -29,16 +29,25 @@ pub trait PalletInfo {
 	fn index<P: 'static>() -> Option<usize>;
 	/// Convert the given pallet `P` into its name as configured in the runtime.
 	fn name<P: 'static>() -> Option<&'static str>;
+	/// Convert the given pallet `P` into its Rust module name.
```

Comment:
maybe we should precise which rust module name. because a rust module is not unique
```suggestion
	/// Convert the given pallet `P` into its Rust module name as used in the runtime construction.
```